src/templates/utils/index.test.ts

```diff
@@ -239,4 +240,14 @@ describe('validating template URLs', () => {
       ).toBe(TEMPLATE_URL_VALID)
     })
   })
+
+  describe('validating template readme Formatter', () => {
```

Comment:
might consider adding another example, maybe with some text below the `## Setup Instructions`.